api/warp17-stats.proto

```diff
@@ -202,6 +202,13 @@ message TcpStatistics {
     required uint32 ts_failed_data_clone = 16;
 
     required uint32 ts_reserved_bit_set  = 17;
+
+    required uint32 ts_sent_syn = 18;
```

Comment:
There is a mention in the commit log of the second commit in this pull request chain.

I added a check in the unit test for the SYN received count. No way to check for FINs since the test criteria is based on runtime so no FINs are sent. Also no way to check for RSTs either. FWIW, for my testing I'm paying close attention to the TCP SYN/FIN/RST pkts and the stats are incrementing as expected.